docs/en/mkdocs.yml

```diff
@@ -6,7 +6,7 @@ theme:
   name: material
   custom_dir: ../en/overrides
   palette:
-  - media: "(prefers-color-scheme)"
+  - media: (prefers-color-scheme)
```

Comment:
Typically, there shouldn't be any changes to `docs/en/mkdocs.yml` when contributing a translation page for another language. Can you revert these, to allow easier reviewing of the Romanian part?

Reply:
So yes, I would expect the link to be added, but I'm confused why other formatting has happened, and comments have been removed etc.

Reply:
This has now been fixed with https://github.com/fastapi/fastapi/pull/13542 😉